tests/storage/xlease_test.py

```diff
@@ -64,16 +66,14 @@ def pwrite(self, offset, buf):
 @pytest.fixture(
     scope="module",
     params=[
-        userstorage.PATHS["file-512"],
-        userstorage.PATHS["file-4k"],
+        BACKENDS["file-512"],
+        BACKENDS["file-4k"],
     ],
     ids=str,
 )
 def user_storage(request):
-    storage = request.param
-    if not storage.exists():
-        pytest.xfail("{} storage not available".format(storage.name))
-    return storage
+    with request.param:
+        yield request.param
```

Comment:
Again this removes the explaining variable (storage) - this change is not required.
I think using "backend" like most test will help to make the code easier to work with.